CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+- Made some updates to TUTORIAL.md to more closely match the current layout of Spotify's Developer Dashboard and the process for creating an app
+
+### Added
+- Added directions to install Spotipy in TUTORIAL.md (pip install spotipy --upgrade)
+
```

Comment:
Please add these points to the already existing headers below.

Reply:
You did what I asked for, but we've released a new version, so this file got changed once again.

We are already discussing this internally about how we are going forward with the CHANGELOG.md file, as the changes can be automatically gathered by GitHub. So we'll ignore this for now until this is resolved.